controllers/front/notify.php

```diff
@@ -153,4 +161,15 @@ public function processValidation()
         $this->message = 'Bad request';
         $this->error = true;
     }
+
+    /**
+     * @param Cart $cart
+     *
+     * @return bool
+     */
+    public function checkOrderExists(Cart $cart)
+    {
+        sleep(rand(1, 5));
```

Comment:
why rand?

Reply:
We are having a issue: 

browsers sometime does redirect 2 times so the function checkOrder is not working and some orders are duplicated, normally you should do a queue but I cant find to find better solution than this "for the moment".